### PR TITLE
Edits for clarity and formatting

### PR DESCRIPTION
**Pull request by mjgutermuth**
Some small edits to use more direct language. The original wording "pod with the lower value will come first" is unclear, so I changed this to "is deleted first" but I'm not 100% sure that's correct.

I added backticks to "using the matching operator `<operator>.*`" because it didn't look like it was rendering correctly on the page.

### CODE REVIEW

File: content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md [hunk 1]
```diff
@@ -1824,7 +1824,7 @@ PodStatus represents information about the status of a pod. Status may trail the
 
 - **phase** (string)
 
-  The phase of a Pod is a simple, high-level summary of where the Pod is in its lifecycle. The conditions array, the reason and message fields, and the individual container status arrays contain more detail about the pod's status. There are five possible phase values:
+  The phase of a Pod is a high-level summary of where the Pod is in its lifecycle. The conditions array, the reason and message fields, and the individual container status arrays contain more detail about the pod's status. There are five possible phase values:
```
Comment (Sea-n) [inline content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md:1827]: I don't think the word "simple" here should be removed.

Comment (mjgutermuth) [reply content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md:1827]: I don't think "simple" is adding anything useful here, but ymmv 🤷🏼‍♀️

Comment (tengqm) [reply content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md:1827]: This file was auto-generated from the upstream source code. Please fix it in the kubernetes/kubernetes repo.

File: content/en/docs/concepts/workloads/controllers/replicaset.md [hunk 2]
```diff
@@ -311,19 +311,18 @@ assuming that the number of replicas is not also changed).
...
-When scaling down, the ReplicaSet controller chooses which pods to delete by sorting the available pods to
-prioritize scaling down pods based on the following general algorithm:
+When scaling down, the ReplicaSet controller chooses which pods to delete based on the following general algorithm:
 
-1. Pending (and unschedulable) pods are scaled down first
+1. Pending and unschedulable pods are scaled down first.
 1. If `controller.kubernetes.io/pod-deletion-cost` annotation is set, then
-   the pod with the lower value will come first.
-1. Pods on nodes with more replicas come before pods on nodes with fewer replicas.
+   the pod with the lower value is deleted first.
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/replicaset.md:321]:
```suggestion
   the pod with the lower value is selected for deletion first.
```

File: content/en/docs/concepts/workloads/controllers/replicaset.md [hunk 3]
```diff
@@ -311,19 +311,18 @@ assuming that the number of replicas is not also changed).
...
+1. Pods on nodes with more replicas are deleted before pods on nodes with fewer replicas.
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/replicaset.md:322]:
```suggestion
1. Pods on nodes with more replicas are selected for deletion ahead of pods on nodes that hold
   fewer replicas.
```

File: content/en/docs/concepts/workloads/controllers/replicaset.md [hunk 4]
```diff
@@ -311,19 +311,18 @@ assuming that the number of replicas is not also changed).
...
 1. If the pods' creation times differ, the pod that was created more recently
-   comes before the older pod (the creation times are bucketed on an integer log scale
-   when the `LogarithmicScaleDown` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled)
+   is deleted before the older pod. The creation times are bucketed on an integer log scale
+   when the `LogarithmicScaleDown` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled.
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/replicaset.md:325]:
```suggestion
1. The ReplicaSet controller prefers to delete the Pods that have been running for
   the **least** amount of time. When the `LogarithmicScaleDown`
   [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is
   enabled in your cluster, the ReplicaSet controller does not use the exact age of
   the Pod. Instead, the controller calculates a score for how old the Pods in the
   ReplicaSet are, using a logarithmic bucketing algorithm. The ReplicaSet controller
   then looks for the lowest score from that algorithm, and selects at random one Pod
   with that score.
```

**Review by sftim — COMMENTED** [review]
`content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md` is a generated page. See the advice at https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#auto-generated-edit-notice

I also have some other feedback.

File: content/en/docs/concepts/workloads/controllers/replicaset.md (hunk 4 above)
Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/replicaset.md:326]: You're welcome to make equivalent changes locally or via the web editor.

Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/replicaset.md:326]: If you'd like more help, join Slack - see https://slack.k8s.io/ - and say Hi in the #sig-docs channel.

**Review by sftim — COMMENTED** [review]
We shouldn't make any changes to https://k8s.io/docs/reference/kubernetes-api/workload-resources/pod-v1/ other than by either:
- tweaking the generator code
- tweaking the Kubernetes API definitions that the generator uses as its input.


https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#auto-generated-edit-notice has some advice on this; it's similar to what I wrote there.

I suggest leaving those changes out of this PR. @mjgutermuth do you know how to squash commits? (if not, we can get GitHub to do something automatically)

File: content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md:1827]: Removing “simple” from the API definition is OK by me. It's also OK to _file an issue_ about removing that word (with some justification).

**Comment by divya-mohan0209** [comment]
Hello @mjgutermuth , gentle reminder on the above PR. Would you be able to incorporate the above suggestions?

**Comment by kbhawkey** [comment]
Hello @mjgutermuth . This pull request has been open for some time. Are you actively working on the changes?

**Comment by kbhawkey** [comment]
Hi @mjgutermuth .
I am closing this pull request. When you have time, you are welcome to reopen the PR and continue to work on the changes.
/close

**Pull request closed without merging**